### ISSUE TITLE
Support srcset and picture 

### ISSUE THREAD

**Issue by thomaskrakow** [vvvmax/unitegallery#231]
It would be great if Unite Gallery support the srcset and sizes attributes and the picture element for full responsive functionality.

srcset and sizes:
https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images

Why support picture element:
https://css-tricks.com/using-webp-images/

**Comment by rustykowski** [comment]
I agree. The srcset attribute in the original markup is currently omitted in the rendered DOM.
